packages/block-editor/src/hooks/text-align.js

```diff
@@ -70,11 +71,17 @@ function BlockEditTextAlignmentToolbarControlsPure( {
 	name: blockName,
 	setAttributes,
 } ) {
+	const isDisabled = useIsTextAlignDisabled( blockName );
```

Comment:
I think we might be able to use `useBlockSettings` here instead of creating a new hook.